ubuntu-headless-qt/README.md

```diff
@@ -0,0 +1,996 @@
+
+The below section shows how to configure for each GPIO function:
+
+#### GPIO
+
+System uses /sys/class/gpio to control the GPIO pin, please refer to the following table:
+
+```
+/-----------|---------------|-------|-----|-----------|-----|-------|---------------|-------------\
+|   pinum   |   Function    | group | pin |   J3 PIN  | pin | group |   Function    |   pinum     |
+|-----------|---------------|-------|-----|-----------|-----|-------|---------------|-------------|
+|           |   3.3V        |       |     |   1   2   |     |       |   5V          |             |
+|   490     |   RIIC3 SDA   |   46  |  2  |   3   4   |     |       |   5V          |             |
+|   491     |   RIIC3 SCL   |   46  |  3  |   5   6   |     |       |   GND         |             |
+|   304     |   GPIO        |   23  |  0  |   7   8   |  0  |   38  |   SCIF0 TX    |   424       |
+|           |   GND         |       |     |   9   10  |  1  |   38  |   SCIF0 RX    |   425       |
+|   456     |   GPIO        |   42  |  0  |   11  12  |  2  |   7   |   GPIO        |   178       |
+|   336     |   GPIO        |   27  |  0  |   13  14  |     |       |   GND         |             |
+|   345     |   GPIO        |   28  |  1  |   15  16  |  0  |   8   |   GPIO        |   184       |
+|           |   3.3V        |       |     |   17  18  |  0  |   15  |   GPIO        |   240       |
+|   465     |   RSPI0 MOSI  |   43  |  1  |   19  20  |     |       |   GND         |             |
+|   466     |   RSPI0 MISO  |   43  |  2  |   21  22  |  1  |   14  |   GPIO        |   233       |
+|   464     |   RSPI0 CK    |   43  |  0  |   23  24  |  3  |   43  |   RSPI0 SSL   |   467       |
+|           |   GND         |       |     |   25  26  |  1  |   11  |   GPIO        |   209       |
+|           |   RIIC0 SDA   |       |     |   27  28  |     |       |   RIIC0 SCL   |             |
+|   152     |   GPIO        |   4   |  0  |   29  30  |     |       |   GND         |             |
+|   153     |   GPIO        |   4   |  1  |   31  32  |  0  |   32  |   GPIO        |   376       |
+|   297     |   GPIO        |   22  |  1  |   33  34  |     |       |   GND         |             |
+|   457     |   CAN0 TX     |   42  |  1  |   35  36  |  1  |   23  |   GPIO        |   305       |
+|   208     |   CAN0 RX     |   11  |  0  |   37  38  |  0  |   46  |   CAN1 TX     |   488       |
+|           |   GND         |       |     |   39  40  |  1  |   46  |   CAN1 RX     |   489       |
+\-----------|---------------|-------|-----|-----------|-----|-------|---------------|-------------/
+```
+
+pinum = $group * $groupin + $pin + $pinbase (where pinbase=120, groupin=8)
+
+Example for J3 PIN 7:
+                              23*8 + 0 + 120 = 304 = pinum
+
+To set GPIO pin, move to GPIO sysfs directory and set values as shown below:
+
+```
+root@rzpi:~# cd /sys/class/gpio/
```

Comment:
I saw from your verification logs. Sometimes, the prompt is 
`root@localhost:~#`
or 
`rzsbc@localhost:`

There is no
`root@rzpi:~#`

Which one is correct?

Reply:
`rzsbc@localhost` is the correct one, I will change from `root@rzpi` to `rzsbc@localhost`